CONTRIBUTING.md

```diff
@@ -185,7 +185,14 @@ If you are updating tests and just want to run a single test to check it, you
 can use this syntax to run it exactly as the test harness would:
 
 ```text
-$ python tools/test.py -v --mode=release parallel/test-stream2-transform
+$ python tools/test.py --mode=release parallel/test-stream2-transform
```

Comment:
>Other option, a wrapper script test-ci (.sh & .cmd).

Doesn't that duplicate the `Makefile` and `vcbuild.bat` `test-ci` target? What's the gain?